src/test/java/com/cinchapi/common/base/AnyObjectsTest.java

```diff
@@ -15,12 +15,9 @@
  */
 package com.cinchapi.common.base;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.TreeMap;
-import java.util.TreeSet;
+import java.util.*;
```

Comment:
No static imports

Reply:
You mean wildcard import? And alright, this was something Intellij automatically did. I think if I auto import something and it can reduce the amount of imports it generally does, I'll have to find out how to disable this feature.